sensor_msgs/include/sensor_msgs/distortion_models.hpp

```diff
@@ -31,6 +31,9 @@
 // ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE
 // POSSIBILITY OF SUCH DAMAGE.
 
+// This file is originally ported from ROS1:
```

Comment:
yes or "this file was originally copied from". Not sure about the exact wording but I think a more generic phrasing will allow for more reuse throughout the codebase